src/lib.rs

```diff
@@ -22,17 +22,20 @@
 
 #![deny(missing_docs)]
 
+extern crate cpp_demangle;
 extern crate gimli;
 extern crate memmap;
 extern crate object;
 extern crate owning_ref;
 extern crate fallible_iterator;
+extern crate rustc_demangle;
 #[macro_use]
 extern crate error_chain;
 
 use object::Object;
 use owning_ref::OwningHandle;
 use fallible_iterator::FallibleIterator;
+use rustc_demangle::demangle;
```

Comment:
We should either do a `use cpp_demangle::Symbol` here, or we should remove the `use` for `rustc_demangle`.